changelogs/unreleased/gh-6433-msgpack-encode-error-cfg.md

```diff
@@ -0,0 +1,5 @@
+## feature/core
```

Comment:
`The new option is enabled by default.` - I am afraid it can't be done like that. You are going to break the existing users who might already encode error objects as strings using msgpack module. Options changing the visible behaviour need to be turned off by default. At least until some ultra breaking Tarantool version is released where we will break everything and drop the legacy.

If you still want to push for that, you need to consult Mons and Dmitry as product managers. Personally I like to drop legacy, but this really can break something.

Reply:
Replied in [this](https://github.com/tarantool/tarantool/pull/6435#discussion_r709616952) thread.